react-components/src/apps/admin/AppStats.js

```diff
@@ -211,22 +294,28 @@ class AppStats extends Component {
                         </Toolbar>
                     </div>
                     <Table className={classes.statTable}>
-                        <TableHead>
+                        <TableHead className={classes.statTableHead}>
                             <TableRow hover>
-                                <TableCell
-                                    className={classes.statTableHead}>{getMessage("appName")}</TableCell>
-                                <TableCell className={classes.statTableHead}
-                                           numeric>{getMessage("rating")}</TableCell>
-                                <TableCell className={classes.statTableHead}
-                                           numeric>{getMessage("total")}</TableCell>
-                                <TableCell className={classes.statTableHead}
-                                           numeric>{getMessage("completed")}</TableCell>
-                                <TableCell className={classes.statTableHead}
-                                           numeric>{getMessage("failed")}</TableCell>
-                                <TableCell className={classes.statTableHead}
-                                           numeric>{getMessage("lastCompleted")}</TableCell>
-                                <TableCell className={classes.statTableHead}
-                                           numeric>{getMessage("lastUsed")}</TableCell>
+                                {columnData.map(column => (
+                                    <TableCell key={column.name}
+                                               numeric={column.numeric}
+                                               sortDirection={orderBy === column.name ? order : false}
+                                    >
+                                        <Tooltip
+                                            title="Sort"
```

Comment:
Should this string be in a messages file for i18n?

Reply:
Yes, good catch 👍